File: ts-space-shooter-starter-09-points/src/player.ts
export class Player{
    name: string;
    score: number;
    constructor(name: string, score: number) {
        this.name = name;
        this.score = score;
    }
    comparer(p1: Player,p2: Player){
        return p2.score-p1.score;
    }
}

File: SpaceShooter/SpaceShooter/Player.cs
using System;

namespace SpaceShooter
{
    public class Player
    {
        public Player(string name, int score)
        {
            Name = name;
            Score = score;
        }

        public String Name { get; set; }

        public int Score { get; set; }
        
    }
}
